Detectors/TPC/base/include/TPCBase/CDBInterface.h

```diff
@@ -123,8 +123,9 @@ class CDBInterface
   /// Return any CalPad object
   ///
   /// The function returns the CalPad object stored at the given path in the CCDB
-  /// \return CalPad object
-  const CalPad& getCalPad(const std::string_view path);
+  /// \return object
+  template <typename T>
+  T& getCalPad(const std::string_view path, long timestamp = -1, std::map<std::string, std::string> metaData = std::map<std::string, std::string>());
```

Comment:
What is the difference to `getSpecificObjectFromCDB`? looks like the exact same function. Perhaps one can remove this and simply make `getSpecificObjectFromCDB` public.

Reply:
I agree, the getCalPad function can be removed. I also implemented the comments below in the latest pushed version.